.clj-kondo/config.edn

```diff
@@ -7,7 +7,12 @@
   :unresolved-var {:exclude [frontend.util/node-path.basename
                              frontend.util/node-path.dirname
                              frontend.util/node-path.join
-                             frontend.util/node-path.name]}}
+                             frontend.util/node-path.name]}
+
+  :consistent-alias
+  {:aliases {datascript.core d
```

Comment:
I made these consistent as I wanted to quickly grep for datascript usage to see if anything would be impacted by upgrading datascript. I mistakenly thought we were upgrading from 1.0.1 but I now see we're upgrading from 1.3.2. Regardless, consistent aliasing is still helpful for code cleanliness

Reply:
So many gems in clj-kondo!